Embark Anyone (#1260)
DFHack/scripts: docs/embark-anyone.rst

embark-anyone
=============

.. dfhack-tool::
    :summary: Allows you to embark as any civilisation, including dead and non-dwarven ones.

>     :tags: embark gameplay armok

Reviewer: TODO: think about the implications of combining `gameplay` and `armok`

(If I submit this review without finishing my thoughts here, please poke me about this)

Reviewer: more words:

I've been treating "gameplay" and "armok" as mutually exclusive, with "gameplay" meaning "changes the gameplay in some consistent way that may not be achievable in vanilla but is not inherently "cheaty" or game-breaking" and armok meaning "no restrictions -- the world conforms to your will".

In other words, "gameplay" tools can have some "armoky" elements to them, but they are intended to be a balanced extension to the world.

This script straddles that line closer than most other tools.

Both `unretire-anyone` and `gui/embark-anywhere` have the `armok` tag and not the `gameplay` tag, so my suspicion is that we should go with `:tags: embark armok` here and remove the `gameplay` tag.

Author: Given that some civs are borderline unplayable, I'll agree that armok alone is appropriate.

> 

Reviewer: Please add an introductory paragraph before the Usage section that at least goes over when to run the command

embark-anyone
========

.. dfhack-tool::
    :summary: Allows you to embark as any civilisation, including dead and non-dwarven ones.
    :tags: embark gameplay armok

Usage
-----

::

    embark-anyone

Note
-----
Non-dwarven civs have their own mechanics which can render fortress mode difficult
to unplayable. Preparing carefully is advised, and cheating in some items may be

> required.

Reviewer 2: i'd avoid the pejorative "cheating" entirely and instead say something like "without spawning them in manually" or some other phrase that describes the action neutrally

don't embed judgments about how people choose to play the game into the documentation

> required.

Author: How about "some crucial items may need to be spawned in"?

embark-anyone
=============

.. dfhack-tool::
    :summary: Allows you to embark as any civilisation, including dead and non-dwarven ones.
    :tags: embark armok

> This script can be ran on the embark screen and presents a list of civilisations that
> are not normally playable to add to the "Select Origin Civ" menu. This includes dead
> civilisations, the normal non-dwarven civilisations, and subterranean animal people.

Reviewer: ```suggestion
This script must be run on the embark screen when choosing an origin civilization.
When run, you can add any civilization, including dead or non-dwarven civilizations,
to the list of choices.
```

Usage
-----

::

    embark-anyone

Note
-----
Non-dwarven civs have their own mechanics which can render fortress mode difficult

> to unplayable. Preparing carefully is advised, and some crucial items may need to be spawned in.

Reviewer: ```suggestion
or unplayable. Preparing carefully is advised, and some crucial items may need to be
spawned in with other DFHack tools (e.g. `gui/create-item`).
```
